core/src/impl/Kokkos_Error.cpp

```diff
@@ -51,6 +51,7 @@
 #include <iomanip>
 #include <stdexcept>
 #include <impl/Kokkos_Error.hpp>
```

Comment:
Remove that include

Reply:
this is a `cpp` file, so it's not that important, but I'm not sure which include you're referring to.

Reply:
`<impl/Kokkos_Error.hpp>`

Reply:
I don't understand. `<impl/Kokkos_Error.hpp>` is used throughout this file (for instance, `Experimental::RawMemoryAllocationFailure` is there).

Reply:
https://github.com/kokkos/kokkos/blob/5b791388dcd3abcd199b2af76c9a0f440cd956f3/core/src/Cuda/Kokkos_Cuda_Error.hpp#L51